### DISCUSSION TITLE
Clothes must have pockets

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#11267] [Ideas and suggestions]
It seems completely wrong to me that clothes other than Medic clothes have no pockets.
Because of this, I dress everyone in Medic clothes. The three extra pockets are very useful.
For example, the clothes of a security officer must have at least 4 pockets for ammunition, grenades and medicines.
In our reality military vest has at least 6-8 pockets.
Mechanic and engineer clothes must have 2-4 pockets for tools, wires, microcircuits and other useful things.

**Comment by bookslayer10** [comment]
Inventory spaces are very powerful. I agree that there should be other clothing with slots in it, but I think that they should be different from then normal ones. They would be crafted with high enough skill instead of just spawning with new players.

**Comment by duarm** [comment]
you can think those bottom slots you always have as your pockets, even tho they don't disappear when you remove your clothes.

**Comment by aeondez** [comment]
Medics in particular need a lot more stuff, but clothes pockets should overall be balanced. 

Clothes with pockets should provide zero bonuses, and have restrictions on which items are allowed to be stored in them.

**Reply by adamkad1** [reply]
I dont think pockets would make clothes so thin as to not have protection or stuff like that

**Comment by CurePindal** [comment]
I played the game when we could almost limitless stack containers into the toolbelt slots. 
Its better and more interesting this way.
So I would disagree - the current inventory system is perfectly suitable!

**Reply by aeondez** [reply]
As someone who has been playing for a very long time, I agree with this. There was a time when the meta was to stack an inventory almost completely full of toolboxes, which were in turn filled to the brim with stuff, as they had 12 slots per toolbox. Then they were stackable inside crates, which made it a nightmare to sort.

There are mods for people who want larger stack sizes, but I don't think the vanilla game should be encouraging hoarding hundreds of items on your person at all times.

**Comment by MateusJK15** [comment]
This could work by restricting what can be stored in those pockets
medical clothes - medicine
captain and security - ammo
engineer and mechanic - tools

**Reply by aeondez** [reply]
I completely agree.

I'd argue that the toolbelt, bandolier, and medic's fatigues already exist for this, but they don't enforce what you can put inside, which sort of negates the whole purpose.

**Reply by CurePindal** [reply]
Nobody wants most of that stuff in pockets. tool, ammo, medicine is the stuff an experienced players keeps in the bar, so its more easy accessable.

**Reply by aeondez** [reply]
Sounds like the only things you'd put in pockets are niche items that aren't constantly being used.

Meaning you probably want a backpack mod, since you're just trying to get expanded inventory.

**Reply by CurePindal** [reply]
No, I think the current inventory system is spot on.

**Reply by aeondez** [reply]
I must have misunderstood.

I'd still like to see pockets only used for specific items.

**Comment by pavel4win** [comment]
General purpose pockets are not needed.
I think clothes for every profession should have at least 2 pockets for specific items.
For the sake of lore. To make it more interesting to wear exactly the clothes of your profession.
doctor - medicines
soldier, captain - ammunition
engineer, mechanic, assistant - tools.
I'm not advocating increasing the number of slots in your inventory. There are many mods for this.
